### PR TITLE
Updates EGD

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: content/events/2022-10-egd/egd/index.md [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-| 16:00 - 16:30 |   | Coffee Break  |    |
-| 16:30 - 17:30 |   | ELIXIR-Galaxy Community meeting (part 2)  |   |
-
-#### Wednesday, October 5
+| 09:00 - 09:30 |   | Arrival  |   |
+| 09:30 - 10:00 |   | Welcome & General Galaxy Update   |
+| 10:00 - 10:30 |   | Talks and presentations I  |   |
+| 10:30 - 11:00 |   | Coffee Break   |   |
+| 11:00 - 12:30 |   | Talks and presentations II |   |
+| 12:00 - 13:30 |   | Lunch Break  |  |
```
Comment (hrhotz) [inline content/events/2022-10-egd/egd/index.md:14]: lunch break is from 12:30 onward, isn't-it?

File: content/events/2022-10-egd/egd/index.md [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
-| 11:30 - 12:00 |   | Coffee Break |   |
-| 12:00 - 13:00 |   | talks and presentations IV  |   |
-| 13:00 - 14:00 |   | Lunch Break  |   | 
-| 14:00 - 16:00 |   | Birds of a Feather  |   |
-| 16:00 - 16:30 |   | Coffee Break  |    |
-| 16:30 - 17:30 |   | Birds of a Feather |   |
+| 09:30 - 10:30 |   | Talks and presentations III   |  |
+| 10:30 - 11:00 |   | Coffee Break |   |
+| 11:00 - 12:30 |   | Talks and presentations IV  |   |
+| 12:00 - 13:30 |   | Lunch Break  |   |
```
Comment (hrhotz) [inline content/events/2022-10-egd/egd/index.md:27]: same as above

File: content/events/2022-10-egd/index.md [hunk 3]
```diff
@@ -24,9 +24,9 @@ To minimize travelling (and yes, we recommend taking a train :-) ), we are co-lo
 
 <br>
 
-- **ELIXIR Galaxy Implementation Study (IS2021)** meeting,
-- **European Galaxy Days (EGD)**, talks, discussions and updates, including ELIXIR-Galaxy community meeting, and CoFest,
-- **EuroScienceGateway (ESG)** kick-off, an EU-funded project to build a mature network of national Galaxy servers and Pulsar endpoints in European clouds and HPC centers.
+- [**ELIXIR Galaxy Implementation Study (IS2021)**](/events/2022-10-egd/is2021/index.md) meeting,
```
Comment (hrhotz) [inline content/events/2022-10-egd/index.md:27]: the existing path is: /events/2022-10-egd/elixir-is2021/index.md

File: content/events/2022-10-egd/index.md [hunk 4]
```diff
@@ -49,101 +49,78 @@ Registration is free, but the space is limited, so we recommend to you register
...
         <strong>Lunch Break</strong>
       </td>
     </tr>
     <tr class="text-center">
-      <td style="background-color: DarkOrange; border: 3px solid white;">
-        <strong>ELIXIR IS2021 meeting</strong>
-        <br><br><small>Strengthen Data Management in Galaxy</small>
+      <th style="border: 3px solid white;">14:00<br> - <br>17:30</th>
+      <td style="background-color: #FBB117; border: 3px solid white;">
+        <strong><a href="/events/2022-10-egd/is2021/index.md">ELIXIR IS2021 meeting</a></strong>
```
Comment (hrhotz) [inline content/events/2022-10-egd/index.md:99]: same as above

**Comment by hrhotz** [comment]
I have added a few comments

File: content/events/2022-10-egd/index.md (hunk 3 above)
Comment (hrhotz) [reply content/events/2022-10-egd/index.md:27]: but feel free to change (and delete the current place holder)

**Comment by beatrizserrano** [comment]
> I have added a few comments

very good points, thanks a lot!

**Pull request merged**